### ISSUE TITLE
LP writer creating invalid names for COIN CBC

### ISSUE THREAD

**Issue by d3netxer** [odow/MathOptFormat.jl#67]
In my Julia file I am creating variables. The output .lp file has brackets for the variables and is creating many errors like the following when I am running cbc from the command line:

```
### CoinLpIO::is_invalid_name(): Name x[1] contains illegal character '['
```

Is there any way to fix this? Maybe we can replace the brackets with underscores?

**Comment by odow** [comment]
LPWriter.jl has some logic about handling valid names. See https://github.com/odow/LPWriter.jl/blob/master/src/LPWriter.jl

We should probably do something similar. I haven't finished #32

**Comment by odow** [comment]
Closed by #71
